frontend/test/metabase/scenarios/chart_drill.cy.spec.js

```diff
@@ -0,0 +1,90 @@
+import { signInAsAdmin, restore, popover } from "__support__/cypress";
+
+describe("chart drill", () => {
+  before(restore);
+  beforeEach(signInAsAdmin);
+
+  xit("should allow brush date filter", () => {
+    cy.visit("/question/new");
```

Comment:
I would much prefer this to be a unit test, if at all possible.

Otherwise, we really need to avoid Cypress tests that incidentally test things like the query builder as part of testing something else, otherwise it's going to be a big pain to update all the tests when something about the query builder changes. If you have to keep this as a Cypress test, use `cy.request()` to create the question directly (we should probably add a helper for that in `__support__/cypress` for that)

Reply:
This test was copied over from `release-0.34.x` since I wanted to add another "chart drill" test. It was added in #11916. That branch doesn't have `restore`. I avoided saving the question so I wouldn't need to clean up after the test ran. Though, that does mean there's more boilerplate. The new test from this PR does use `cy.request()`.

I do think this test should be in Cypress since it covers both viz manipulation and updating/rerunning the query. The bug that prompted that was from the `_transformed` flag persisting as new data was fetched. I think that would have been hard to catch with a unit test.

That said, now that it's on a branch with `restore`, I'll updating it to create a new question via the API.

Reply:
👍